pkgs/by-name/li/libbytesize/package.nix

```
@@ -52,6 +53,18 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
+  postInstall = ''
+    substituteInPlace $out/${python3.sitePackages}/bytesize/bytesize.py \
+      --replace 'CDLL("libbytesize.so.1")' "CDLL('$out/lib/libbytesize.so.1')"
+  '';
+
+  passthru.tests = {
+    python = runCommand "${finalAttrs.pname}-test-python" { } ''
+      ${python3.withPackages (ps: [ ps.bytesize ])}/bin/python -c 'import bytesize'
+      touch $out
+    '';
+  };
```

```suggestion
  pythonImportsCheck = [ "bytesize" ];
```

(Add `pythonImportsCheckHook` to `nativeBuildInputs`)

---

This doesn't look right. `pythonImportChecks` is mentioned exactly once in all of nixpkgs. Maybe it has been renamed?

---

Oh, it's `pythonImportsCheck`, not `pythonImportChecks`.